### ISSUE TITLE
tests/store.test.js is not executed

### ISSUE THREAD

**Issue by FezVrasta** [mxstbr/react-boilerplate#458]
When I run `npm test` I get:

```Shell
START:
ERROR: 'Warning: Failed propType: Required prop `href` was not specified in `A`.'
  <A />
    ✔ should render its children
ERROR: 'Warning: Failed propType: Required prop `children` was not specified in `A`.'
    ✔ should adopt the className
    ✔ should adopt the href
    ✔ should adopt the target
  <Button />
    ✔ should render its children
ERROR: 'Warning: Failed propType: Required prop `children` was not specified in `Button`.'
    ✔ should adopt the className
    ✔ should render an <a> tag if no route is specified
    ✔ should render a button to change route if the handleRoute prop is specified
    ✔ should handle click events
  <Footer />
    ✔ should render the copyright notice
    ✔ should render the credits
  <H1 />
    ✔ should render its text
  <H2 />
    ✔ should render its text
  <H3 />
    ✔ should render its text
  <Img />
    ✔ should render an <img> tag
    ✔ should have an alt attribute
  <IssueIcon />
    ✔ should render a SVG
  <List />
    ✔ should render the component if no items are passed
    ✔ should render the items
  <ListItem />
    ✔ should adopt the className
    ✔ should render the content passed to it
  <LoadingIndicator />
    ✔ should render 14 divs
  App Actions
    loadRepos
      ✔ should return the correct type
    reposLoaded
      ✔ should return the correct type and the passed repos
    repoLoadingError
      ✔ should return the correct type and the error
ERROR: 'Warning: Failed propType: Invalid prop `src` of type `object` supplied to `Img`, expected `string`.'
  <App />
    ✔ should render the logo
    ✔ should render its children
    ✔ should render the footer
  appReducer
    ✔ should return the initial state
    ✔ should handle the loadRepos action correctly
    ✔ should handle the reposLoaded action correctly
    ✔ should handle the repoLoadingError action correctly
  selectGlobal
    ✔ should select the global state
  selectCurrentUser
    ✔ should select the current user
  selectLoading
    ✔ should select the loading
  selectError
    ✔ should select the error
  selectRepos
    ✔ should select the repos
  selectLocationState
    ✔ should select the route as a plain JS object
  <FeaturePage />
    ✔ should render its heading
    ✔ should link to "/"
  Home Actions
    changeUsername
      ✔ should return the correct type and the passed name
  <HomePage />
    ✔ should render the loading indicator when its loading
    ✔ should render an error if loading failed
    ✔ should render the repositories if loading was successful
    ✔ should link to /features
  homeReducer
    ✔ should return the initial state
    ✔ should handle the changeUsername action correctly
  getGithubData Saga
    ✔ should dispatch the reposLoaded action if it requests the data successfully
LOG: undefined
    ✔ should call the repoLoadingError action if the response errors
  selectHome
    ✔ should select the home state
  selectUsername
    ✔ should select the username
  <NotFound />
    ✔ should render the Page Not Found text
    ✔ should render a button
    ✔ should link to "/"
  <RepoListItem />
    ✔ should render a ListItem
    ✔ should not render the current username
    ✔ should render usernames that are not the current one
    ✔ should render the repo name
    ✔ should render the issue count
    ✔ should render the IssueIcon
  hooks
    getHooks
      ✔ given a store, should return all hooks
    helpers
      injectAsyncReducer
        ✔ given a store, it should provide a function to inject a reducer
      injectAsyncSagas
        ✔ given a store, it should provide a function to inject a saga
  request
    stubbing successful response
      ✔ should format the response correctly
    stubbing error response
      ✔ should catch errors

=============================== Coverage summary ===============================
Statements   : 98.58% ( 1184/1201 ), 42 ignored
Branches     : 95.68% ( 354/370 ), 64 ignored
Functions    : 95.97% ( 262/273 ), 6 ignored
Lines        : 98.41% ( 745/757 )
================================================================================

Finished in 0.193 secs / 0.112 secs

SUMMARY:
✔ 65 tests completed
```

In this output, I don't see the tests defined in `store.test.js`.

Why aren't included?

**Comment by FezVrasta** [comment]
I guess the problem is inside the test-bundler, the regex excludes store.test.js

**Comment by mxstbr** [comment]
Good catch, we should definitely add that!

**Comment by mxstbr** [comment]
Fixed by the amazing @uzarubin in #858!
